src/main/java/edu/ucdavis/dss/ipa/services/jpa/JpaScheduleOpsService.java

```diff
@@ -173,4 +172,169 @@ private String adjustTermCodeYear(String termCode, Long toYear) {
 
 		return newTermCode;
 	}
+
+	/**
+	 * Syncs CRN and location data from DW to IPA, assuming the section/activities already exist
+	 */
+	@Transactional
+	@Override
+	public void updateSectionsFromDW() {
+
+		List<Course> courses = this.courseService.getAllCourses();
+
+		// Map Keys will look like allDwSections.get(PSC-2017);
+		Map<String, List<DwSection>> allDwSections = new HashMap<>();
+
+		for (Course course : courses) {
+
```

Comment:
for loops shouldn't begin with blank lines (scopes generally shouldn't start with blank lines)